### ISSUE TITLE
rel between "axonemal dynein complex" and "cytoplasmic dynein complex"

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11830]
Hi,

It looks quite odd to me that "axonemal dynein complex" is_a "cytoplasmic dynein complex".

I see that the term "axoneme part" is_a "cytoplasmic part", and perhaps this is part of the reason.

However, the people who study dyneins would not consider an axonemal dynein to be a cytoplasmic dynein. Phylogenetic trees of dyneins separate into three groups:
- cytoplasmic dyneins (the "regular" one(s) that move on cytoplasmic MTs
- axonemal inner arm dyneins
- axonemal outer arm dyneins

So, maybe we could give "axonemal dynein complex" direct parentage under "dynein complex" GO:0030286

Maybe we don't need the "axoneme part" is_a "cytoplasmic part" relationship either, since teh axoneme is part of the cilium, which is separate from the cytoplasm.

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/11666](https://sourceforge.net/p/geneontology/ontology-requests/11666)

**Comment by paolaroncaglia** [comment]
Hi Karen and Helen,

Thanks for your patience. It looks like we already discussed the issue of the axoneme being (or not being) cytoplasmic. Karen raised it in a SF ticket and then addressed it in an email thread involving cilia experts. For brevity, I’ll link the original SF ticket below, and will copy the resolution following experts’ feedback; I will email you both the last message on the thread in case you wish to review the whole discussion.

Original ticket (now on github): 
https://github.com/geneontology/go-ontology/issues/10901

Resolution from email thread: 
(Karen wrote) “Although I previously said I was leaning towards classifying the cilium as a cytoplasmic structure that is bounded by the ciliary membrane and which protrudes through the plasma membrane, David has reminded me that the ciliary membrane contains transmembrane receptors that we do not want to classify as "cytoplasmic" in GO. Therefore, we will make the "ciliary membrane" a specialized part of the "plasma membrane" and then the rest of the cilium will be cytoplasmic, but the entire cilium will not be classified as "cytoplasmic”.”

It is based on that discussion that the axoneme figures as cytoplasmic. Here’s the full stanza for ’axonemal dynein complex’:

[Term]
id: GO:0005858
name: axonemal dynein complex
namespace: cellular_component
def: "A dynein complex found in eukaryotic cilia and flagella; the motor domain heads interact with adjacent microtubules to generate a sliding force which is converted to a bending motion. May contain two or three dynein heavy chains as well as several light chains." [ISBN:0815316194]
synonym: "axonemal dynein heavy chain" NARROW []
synonym: "axonemal dynein intermediate chain" NARROW []
synonym: "axonemal dynein intermediate light chain" NARROW []
synonym: "axonemal dynein light chain" NARROW []
is_a: GO:0005868 {is_inferred="true"} ! cytoplasmic dynein complex
is_a: GO:0044447 {is_inferred="true"} ! axoneme part
intersection_of: GO:0030286 ! dynein complex
intersection_of: part_of GO:0005930 ! axoneme

So the ‘dynein complex’ parentage is already there, but because it’s coupled to part_of axoneme, the link to cytoplasm holds. And we can’t *not* say that ‘axonemal dynein complex’ is part of the axoneme… suggestions? Karen, am I missing any discussion on this topic we may have had more recently?

Thanks,

Paola

**Comment by paolaroncaglia** [comment]
@krchristie

**Comment by paolaroncaglia** [comment]
@hattrill
Hi Helen, please let me know if this username works, thanks!
Paola

**Comment by krchristie** [comment]
Hi Paola,

I think this is largely a nomenclature issue, in that researchers who study dyneins use the phrase "cytoplasmic dynein" to refer to the ones that are present in the cytosol as distinct from an "axonemal dynein" which is found in the cilium/flagellum.

Could we just move "axonemal dynein complex" so that it is a direct child of "dynein complex" and thus a sibling of "cytoplasmic dynein complex", instead of a child of it. We might need to add to either the definition or comment of "cytoplasmic dynein complex" to state explicitly that this term does NOT encompass axonemal dyneins. Otherwise, it looks to me like we might not need to do much else. The term "axoneme" already has an "is_a" relationship to "cytoplasmic part" and a "part_of" relationship to "ciliary cytoplasm", which is consistent with our earlier discussion, so the existing relationships look like they are already adequate if we just move "axonemal dynein complex" to be a direct child of "dynein complex".

Do you think that would work Paola?

-Karen

**Comment by hattrill** [comment]
I agree with Karen. This solution reflects the understanding/use of these terms by the research community.

Thanks, Helen

**Comment by paolaroncaglia** [comment]
Hi Karen and Helen,

I can certainly see your point of wanting to distinguish 'classical' cytoplasmic dynein complex vs. axonemal ones. I could certainly move "axonemal dynein complex" so that it is a direct child of "dynein complex" and thus a sibling of "cytoplasmic dynein complex". However, the link to cytoplasm would still hold because of these logical definitions:

axonemal dynein complex part_of axoneme; axoneme part_of ciliary cytoplasm; ciliary cytoplasm is_a cytoplasm

cytoplasmic dynein complex part_of cytoplasm

The automatic inference script would therefore soon add the link back that axonemal dynein complex is_a cytoplasmic dynein complex. We can't escape from that unless we break the link between axoneme and cytoplasm, which would be contrary to our previous resolution. I'm afraid I cannot think of another fix but I'd be happy to take ideas if you have any. Would it work for you if I added a definition gloss or comment saying that the two classes of dyneins are phylogenetically different? Or any text that you might suggest to that extent. I'm afraid we'd have no way of capturing the difference otherwise. To GO, both complexes are still located in some kind of cytoplasm. It's hard to capture evolution of protein families - afraid that's scope for a different ontology. Also we might add has_part links to any PRO entity if applicable? E.g. axonemal dynein complex has_part [type appropriate protein family X here], cytoplasmic dynein complex has_part [type appropriate protein family Y here].

Let me know what you think, I'd be happy to fix this for you.

Thanks,

Paola

**Comment by krchristie** [comment]
Hi Paola,

I understand your point about the automatic inference regenerating the link.

However, it still seems like there is a flaw here. The correct definition of the "cytoplasmic dynein complex" would include disjoint_from "ciliary cytoplasm", while the axonemal dyneins are only in "ciliary cytoplasm".

-Karen

**Comment by cmungall** [comment]
Could be fixed by having a logical definition using cell body cytoplasm

But ultimately the logical definition involving location may be fake if the true definition is phylogenetic. In which case it would be defined by the fact it contains a particular phylogenetic class of dyneins (with the ID coming from panther or PR). We could still have subclass partof some X axioms where we know these hold universally.

**Comment by hattrill** [comment]
I think that we're loosing the fact that these are very distinctive structures. I'd like to suggest that rather than define the dynein complexes by their localization, we define them by the different structures/functions of the complexes.

-dynein
--non-motile dynein
---axonemal (or motile?) dynein
----inner dynein arm
----outer dynein arm
 

What is traditionally called "cytoplasmic dynein" consists of the cytoplasmic dyneins with their many functions: organelle positioning, microtubule transport, kinechore positioning, force/tension generating, etc. They also include a ciliary dynein, the intraflagellar transport dynein (sometimes called cytoplasmic dynein 2). "Cytoplasmic" dynein complexes are homodimers composed of 2 heavy chains, 2 intermediate chains, 2 light intermediate chain and 2x 3 types of light chain.
To move away from the "cytoplasmic" tag, we could call label them "non-motile dynein" OR "transport dynein"......remove part_of cytoplasm relation (?)

We already have "axonemal dynein complex" .... which is fine as all axonemal dyneins live in axonemal-type structures and the children: inner dynein arm and outer dynein arm.
I think the ending of the definition of "axonemal dynein complex" (A dynein complex found in eukaryotic cilia and flagella...........May contain two or three dynein heavy chains as well as several light chains.) needs to be changed as it is specifically describing the outer dynein arm.

(Outer-arm dynein are heteromeric, made up from two or three heavy chains (alpha, beta +/- gamma) plus about 15 other subunits including various light and intermediate chain.
Inner dynein arms are heteromeric, comprising 8 different heavy chains and various subunits.)

And, to add more to this - I know it's been discussed before, but as the axoneme is a "gated" intracellular structure, I would favor the removal of the is_a cytoplasm part relationship.

**Comment by paolaroncaglia** [comment]
Hi Helen, 

Your ideas make sense to me, and as for your last sentence, it seems to agree with Chris' earlier comments that "Could be fixed by having a logical definition using cell body cytoplasm".

Karen and Chris, please let us know if you have any further comments. @krchristie , @cmungall 

If we have an agreement, I'll implement next week.

Thanks all!
Paola

**Comment by krchristie** [comment]
Hi,

Sorry so long to get to this. I've had a lot going on lately.

I agree that the dynein terms should not be "defined" in terms of location, either in terms of the descriptive definition or the logical one. 

However, regardless of what we name the terms, we are not going to be able to get rid of the name "cytoplasmic dynein complex" as this is the major common use name that is used in the literature; we must keep it at least as a synonym. There are other places in the ontology where we have given preference to the common use name, even when it is not ideal with respect to to the ontology, and I wonder if that is a place where the common use name should remain. If we do change the names, I strongly prefer "transport dynein" over "non-motile dynein" as I'm really not sure what the motile vs non-motile distinction is supposed to refer to for dyneins. If this is meant to refer to whether the cell is motile or not, it doesn't work as cells with motile cilia are not necessarily motile, e.g. tracheal epithelium cells are not motile.

With respect to the structure of the ontology, in her diagram above Helen left the axonemal dynein as a child of what she called "non-motile dynein", like this:

-dynein
--non-motile dynein
---axonemal (or motile?) dynein
----inner dynein arm
----outer dynein arm

I think that the "axonemal dynein" should become a direct child of "dynein" and should NOT remain an is_a child of the term currently called "cytoplasmic dynein" because neither "axonemal dynein" or "cytoplasmic dynein" (by this or any other name) are is_a children of each other. So, I think we need something more like this:

-dynein complex
--cytoplasmic (or transport) dynein complex 
--axonemal dynein complex
----inner arm dynein 
----outer arm dynein 

Regarding whether or not the "ciliary cytoplasm" should be considered part of the "cytoplasm". I am inclined to agree with Helen that this is not correct, despite the fact that we agreed to make it part of the cytoplasm when we discussed this with several ciliary researchers. At the FASEB Cilia and Flagella meeting I attended this summer, there were several talks about the fact that the composition of the both the plasm and the membrane of the cilium is detectably different from that in the non-ciliary cytosol and plasma membrane, e.g. calcium ion concentration, membrane lipid composition, more. By analogy with the "nucleoplasm" which is excluded from "cytoplasm", the cilium is gated by a structure that is basically the same structure as a nuclear pore, so it seems equally valid to exclude the ciliary plasm from the cytoplasm.

However, if we make a change, I think it should be made at the level of the term "ciliary cytoplasm" itself, not just at "axoneme" since I would say the it is the cilium which is the gated structure, not just the axoneme. Note that this change would also require changing the definition of the term "cytoplasm" itself.

-Karen

**Comment by hattrill** [comment]
I agree with all this:

Yep, that's the structure I meant:
-dynein complex
--cytoplasmic (or transport) dynein complex
--axonemal dynein complex
----inner arm dynein
----outer arm dynein 

Sorry, bit of cut & paste magic must of happened!

Cytoplasmic & axonemal dynein complexes are the names I would prefer as it exactly what the community uses.

Helen

**Comment by krchristie** [comment]
Oh good. It sounds like we are exactly on the same page Helen. I too would strongly prefer that we use the names used by the research community, i.e. cytoplasmic dynein & axonemal dynein, and define them correctly without using location as part of the definition.

-Karen

**Comment by paolaroncaglia** [comment]
Hi Karen, Helen and Chris @cmungall 

Thanks for your patience. Looking at this again, I think that the biggest issue is upstream of ‘cilium’, and could be fixed by breaking the link between ‘cell projection cytoplasm’ and ‘cytoplasm’. Karen writes in her last comment that

“Regarding whether or not the "ciliary cytoplasm" should be considered part of the "cytoplasm". I am inclined to agree with Helen that this is not correct, despite the fact that we agreed to make it part of the cytoplasm when we discussed this with several ciliary researchers. At the FASEB Cilia and Flagella meeting I attended this summer, there were several talks about the fact that the composition of the both the plasm and the membrane of the cilium is detectably different from that in the non-ciliary cytosol and plasma membrane, e.g. calcium ion concentration, membrane lipid composition, more. By analogy with the "nucleoplasm" which is excluded from "cytoplasm", the cilium is gated by a structure that is basically the same structure as a nuclear pore, so it seems equally valid to exclude the ciliary plasm from the cytoplasm.”

The situation is similar for plasm of axons and dendrites:

“Axoplasm has a different composition of organelles and other materials than that found in the neuron's cell body (soma) or dendrites. In axoplasmic transport, materials are carried through the axoplasm to or from the soma.” (https://en.wikipedia.org/wiki/Axoplasm) 

Chris, I propose to remove the intersection and links between ‘cell projection cytoplasm’ and ‘cytoplasm’, ‘cytoplasmic part’ and ‘cytoplasmic region’. If so, we’d also better rename ‘cell projection cytoplasm’ and its children as e.g. ‘cell projection plasm’ etc.
We could also add a comment to ‘cell projection plasm’ about the difference in composition from the cell cytoplasm.
Do you think this should be discussed at Thursday’s editors call? Or I may go ahead with the changes, and circulate an email about the new structure once it’s implemented.

The dynein edits would follow.

Thanks,

Paola

**Comment by dosumis** [comment]
Hi Paola,

> Chris, I propose to remove the intersection and links between ‘cell projection cytoplasm’ and ‘cytoplasm’, ‘cytoplasmic part’ and ‘cytoplasmic region’. If so, we’d also better rename ‘cell projection cytoplasm’ and its children as e.g. ‘cell projection plasm’ etc.
> We could also add a comment to ‘cell projection plasm’ about the difference in composition from the cell cytoplasm.

Please don't.  No neurobiologist would consider the 'plasm' of axons and dendrites to not be cytoplasm.  Treating it as something different would cause problems for the synapse project too.

If you think it is necessary to treat the 'plasm' of the cilium differently then the link to cytoplasm should be severed only for these projections.

From Karen's comments, you might also be tempted to change the membrane classificatio/partonomy for ciliary membrane too.  But please note that this would affect some inference.  e.g. 

![image](https://cloud.githubusercontent.com/assets/112839/12417056/37b23cf0-be9d-11e5-8624-ec76a721c380.png)

Inference to 'protein localization to plasma membrane' comes from 
![image](https://cloud.githubusercontent.com/assets/112839/12417067/48726e2a-be9d-11e5-8b64-e7538ebfdd41.png)

Cheers,
David

**Comment by paolaroncaglia** [comment]
Aaargh. Thanks David. The problem is, cilium is_a cell projection, and regardless of what we do with cilium cytoplasm, it would still inherit the link to cytoplasm via cell projection. And I don't feel that we should severe the link btw cilium and cell projection.
Uhm...

**Comment by dosumis** [comment]
label "ciliary cytoplasm"^^string
definition "All of the contents of a cilium, excluding the plasma membrane surrounding the cilium."^^string
EquivalentTo: cytoplasm and 'cell projection cytoplasm' and ('part of' some cilium) # Bit odd!
SubClassOf: 
'cell projection cytoplasm' # inferred
'ciliary part'

ciliary plasm
(exact?)_synonym: ciliary cy
I don't think we need to. Here's the (slightly odd) current formalisation of 'ciliary cytoplasm'

![image](https://cloud.githubusercontent.com/assets/112839/12419541/b75b471a-beae-11e5-846b-298148eb50cd.png)


I'm not sure how you'd classify

**Comment by paolaroncaglia** [comment]
Thanks. So we'd be saying that in general, plasm of cell projections is a subtype of cell cytoplasm, but ciliary plasm would be an exception. Removing the link to projection cytoplasm would ensure that the inference doesn't come back. And no-one could object that *conceptually*, this exception may point to a TPV... correct? :-) (I'd add a def comment to this effect, for future reference)

**Comment by dosumis** [comment]
> So we'd be saying that in general, plasm of cell projections is a subtype of cell cytoplasm, but ciliary plasm would be an exception. 

Kindof. We don't have a general class 'plasm'.   

> Removing the link to projection cytoplasm would ensure that the inference doesn't come back. And no-one could object that conceptually, this exception may point to a TPV... correct? :-) 

There's no incorrect inference via cilium classification as a 'cell projection'.  That would only be an issue if the 'cell projection' class had the axiom: 'has_part some cytoplasm'.

**Comment by paolaroncaglia** [comment]
Brilliant. You'd make a good lawyer David ;-)
Everyone else ok with this proposal?

**Comment by hattrill** [comment]
That sounds good to me. Thanks.

**Comment by krchristie** [comment]
Sounds like this will solve the issue. Thanks.

**Comment by paolaroncaglia** [comment]
Edited 'ciliary cytoplasm' as follows:

OLD STANZA:
[Term]
id: GO:0097014
name: ciliary cytoplasm
namespace: cellular_component
def: "All of the contents of a cilium, excluding the plasma membrane surrounding the cilium." [GOC:BHF, GOC:cilia, PMID:17895364]
comment: Note that we deem cilium and microtubule-based flagellum to be equivalent.
synonym: "cilial cytoplasm" EXACT []
synonym: "cilium cytoplasm" EXACT []
synonym: "microtubule-based flagellar cytoplasm" EXACT []
synonym: "microtubule-based flagellar matrix" NARROW []
synonym: "microtubule-based flagellum cytoplasm" EXACT []
synonym: "microtubule-based flagellum matrix" NARROW []
is_a: GO:0032838 ! cell projection cytoplasm
is_a: GO:0044441 ! ciliary part
intersection_of: GO:0005737 ! cytoplasm
intersection_of: GO:0032838 ! cell projection cytoplasm
intersection_of: part_of GO:0005929 ! cilium

NEW STANZA:
[Term]
id: GO:0097014
name: ciliary plasm
namespace: cellular_component
def: "All of the contents of a cilium, excluding the plasma membrane surrounding the cilium." [GOC:BHF, GOC:cilia, GOC:dos, PMID:17895364]
comment: Note that we deem cilium and microtubule-based flagellum to be equivalent. Also, researchers consider the composition of both the plasm and the membrane of the cilium to be detectably different from that in the non-ciliary cytosol and plasma membrane (e.g. in terms of calcium ion concentration, membrane lipid composition, and more). For this reason, the term "ciliary plasm" is not linked to "cytoplasm".
synonym: "cilial cytoplasm" RELATED []
synonym: "ciliary cytoplasm" RELATED []
synonym: "cilium cytoplasm" RELATED []
synonym: "cilium plasm" EXACT []
synonym: "microtubule-based flagellar cytoplasm" RELATED []
synonym: "microtubule-based flagellar matrix" NARROW []
synonym: "microtubule-based flagellum cytoplasm" RELATED []
synonym: "microtubule-based flagellum matrix" NARROW []
is_a: GO:0044441 ! ciliary part

More soon...

**Comment by paolaroncaglia** [comment]
Note for self: remember, we may want to create a generic ‘plasm’ term, and classify beneath it

**Comment by paolaroncaglia** [comment]
OK, so now I need to fix definition and placement for ‘cytoplasmic dynein complex’ and ‘axonemal dynein complex’.
Here’s my proposal based on Helen’s and Karen’s feedback:

***
cytoplasmic dynein complex

Current stanza:

[Term]
id: GO:0005868
name: cytoplasmic dynein complex
namespace: cellular_component
def: "Any dynein complex that catalyzes movement along a cytoplasmic microtubule; cytoplasmic dynein complexes participates in many transport activities in eukaryotes, such as mRNA localization, intermediate filament transport, nuclear envelope breakdown, apoptosis, transport of centrosomal proteins, mitotic spindle assembly, virus transport, kinetochore functions, and movement of signaling and spindle checkpoint proteins. Subunits associated with the dynein heavy chain mediate association between dynein heavy chain and cargoes, and may include light chains and light intermediate chains." [GOC:mah, PMID:12600311]
synonym: "cytoplasmic dynein heavy chain" NARROW []
synonym: "cytoplasmic dynein intermediate chain" NARROW []
synonym: "cytoplasmic dynein intermediate light chain" NARROW []
synonym: "cytoplasmic dynein light chain" NARROW []
is_a: GO:0030286 {is_inferred="true"} ! dynein complex
is_a: GO:0044444 {is_inferred="true"} ! cytoplasmic part
intersection_of: GO:0030286 ! dynein complex
intersection_of: part_of GO:0005737 ! cytoplasm

- name => stays the same to reflect community usage
- def - leave as is, but
- add def comment: Note that this term is labelled based on phylogenetic classification and community usage, rather than strict cellular localization. Cytoplasmic dynein complexes may contain ciliary dyneins; therefore the term is not linked to ‘cytoplasm’. Cytoplasmic dynein complexes do not contain axonemal dyneins; see GO:0005858 axonemal dynein complex.
- add GOC:hla, GOC:krc
- synonyms - leave as are
- delete link is_a cytoplasmic part
- remove intersections
***

***
2) axonemal dynein complex

Current stanza:

[Term]
id: GO:0005858
name: axonemal dynein complex
namespace: cellular_component
def: "A dynein complex found in eukaryotic cilia and flagella; the motor domain heads interact with adjacent microtubules to generate a sliding force which is converted to a bending motion. May contain two or three dynein heavy chains as well as several light chains." [ISBN:0815316194]
synonym: "axonemal dynein heavy chain" NARROW []
synonym: "axonemal dynein intermediate chain" NARROW []
synonym: "axonemal dynein intermediate light chain" NARROW []
synonym: "axonemal dynein light chain" NARROW []
is_a: GO:0005868 {is_inferred="true"} ! cytoplasmic dynein complex
is_a: GO:0044447 {is_inferred="true"} ! axoneme part
intersection_of: GO:0030286 ! dynein complex
intersection_of: part_of GO:0005930 ! axoneme

Helen wrote:
“I think the ending of the definition of "axonemal dynein complex" (May contain two or three dynein heavy chains as well as several light chains.) needs to be changed as it is specifically describing the outer dynein arm.
(Outer-arm dynein are heteromeric, made up from two or three heavy chains (alpha, beta +/- gamma) plus about 15 other subunits including various light and intermediate chain.
Inner dynein arms are heteromeric, comprising 8 different heavy chains and various subunits.)”

I would remove that last sentence entirely. As for the children, the def of ‘outer dynein arm’ already contains very specific info about composition. If that’s applicable across species, I’ll leave it as is. The def of ‘inner dynein arm’ does not mention composition, but if what Helen writes is applicable across species, I can add it (“Inner dynein arms are heteromeric, comprising 8 different heavy chains and various subunits.”)

Everything else in the stanza can stay as is.
***

Please confirm that I’m reading all this correctly :-) and I’ll make the changes. Thanks!

Paola

**Comment by hattrill** [comment]
Hi Paola,

Axonemal dynein edit sounds correct. I wonder if we could adjust the cytoplasmic dynein complex definition to allude to its structural detinction and role in cillary/flagella transport. Something like:

"Any dynein complex with a homodimeric dynein heavy chain core that catalyzes movement along a microtubule. Cytoplasmic dynein complexes participates in many cytoplasmic transport activities in eukaryotes, such as mRNA localization, intermediate filament transport, nuclear envelope breakdown, apoptosis, transport of centrosomal proteins, mitotic spindle assembly, virus transport, kinetochore functions, and movement of signaling and spindle checkpoint proteins. Some complexes participate in intraflagellar transport. Subunits associated with the dynein heavy chain mediate association between dynein heavy chain and cargoes, and may include light chains and light intermediate chains."

Thanks,

Helen

**Comment by paolaroncaglia** [comment]
Thanks Helen, yes I'll adjust the def as you suggest.

**Comment by paolaroncaglia** [comment]
DONE! And of course I made 'axonemal dynein complex' a sibling of 'cytoplasmic dynein complex' rather than a child of it. Here are the revised stanzas: 

[Term]
id: GO:0005858
name: axonemal dynein complex
namespace: cellular_component
def: "A dynein complex found in eukaryotic cilia and flagella; the motor domain heads interact with adjacent microtubules to generate a sliding force which is converted to a bending motion." [GOC:cilia, GOC:hla, GOC:krc, ISBN:0815316194]
synonym: "axonemal dynein heavy chain" NARROW []
synonym: "axonemal dynein intermediate chain" NARROW []
synonym: "axonemal dynein intermediate light chain" NARROW []
synonym: "axonemal dynein light chain" NARROW []
is_a: GO:0030286 ! dynein complex
is_a: GO:0044447 {is_inferred="true"} ! axoneme part
intersection_of: GO:0030286 ! dynein complex
intersection_of: part_of GO:0005930 ! axoneme

[Term]
id: GO:0005868
name: cytoplasmic dynein complex
namespace: cellular_component
def: "Any dynein complex with a homodimeric dynein heavy chain core that catalyzes movement along a microtubule. Cytoplasmic dynein complexes participate in many cytoplasmic transport activities in eukaryotes, such as mRNA localization, intermediate filament transport, nuclear envelope breakdown, apoptosis, transport of centrosomal proteins, mitotic spindle assembly, virus transport, kinetochore functions, and movement of signaling and spindle checkpoint proteins. Some complexes participate in intraflagellar transport. Subunits associated with the dynein heavy chain mediate association between dynein heavy chain and cargoes, and may include light chains and light intermediate chains." [GOC:cilia, GOC:hla, GOC:krc, GOC:mah, PMID:12600311]
comment: Note that this term is labelled based on phylogenetic classification and community usage, rather than strict cellular localization. Cytoplasmic dynein complexes may contain ciliary dyneins; therefore the term is not linked to 'cytoplasm'. Cytoplasmic dynein complexes do not contain axonemal dyneins; see GO:0005858 axonemal dynein complex.
synonym: "cytoplasmic dynein heavy chain" NARROW []
synonym: "cytoplasmic dynein intermediate chain" NARROW []
synonym: "cytoplasmic dynein intermediate light chain" NARROW []
synonym: "cytoplasmic dynein light chain" NARROW []
is_a: GO:0030286 {is_inferred="true"} ! dynein complex

**Comment by paolaroncaglia** [comment]
I think this is all done, so I'll close this ticket. As for a generic 'plasm' term, not sure what the proper name should be (as plasm is often used for cytoplasm), so let's leave it aside for now. Should the need for it come up again, we'll open a new ticket.

**Comment by hattrill** [comment]
Perfect! Many thanks for your epic efforts. This deserves some sort of reward ....... I'll let you take the weekend off!

**Comment by ValWood** [comment]
That's longer than some of  my tickets. I don't feel so bad now...

**Comment by paolaroncaglia** [comment]
@ValWood , for a split second when I saw your notification I was terrified you wanted to change something :-)

You all have a great weekend!!

**Comment by krchristie** [comment]
Awesome! Thanks for all your work on this Paola.
